dct_backend/dct.py

```diff
@@ -30,3 +31,21 @@ def save_food_item(self, name: str, station: str, meal_time: str,
         item = FoodItem(name, station, meal_time, calories, categories)
         self.menu.append(item)
         return item
+
+    """
+        save_food_station adds a food station to the list of stations
+        name: str
+            The name of the food station
+        
+        returns: FoodStation
+            The food station that was added to the list of food stations
+    """
+    def save_food_station(self, name: str) -> FoodStation:
+        station = FoodStation(name)
+        exists = False
+        for s in self.stations:
+            if (station.name == s.name):
+                exists = True
+        if (exists == False):
+            self.stations.append(station)
+        return station
```

Comment:
For my backend implementation of **Save Seated Section** (which is similar to this), instead of returning the object. I returned a boolean (True for success & False for failure), that way the front end can know if the object was updated properly. 

This might not apply to your code, however, @RafaelPiloto10 suggested this for my code, so he might have more insight about that

Reply:
Oh thats an interesting implementation. It does make more sense to do it like that since this is only saving the stations, not necessarily getting them